### ISSUE TITLE
Apple Intelligence 99% Solution

### ISSUE THREAD

**Issue by hxhlb** [leminlimez/Nugget#160]
Forgive my poor English!

I don’t know if anyone else is stuck at 99% download like me.

**The reason is I turned “Enable Internal Storage” option**

Found this problem because my OTA update is not working
When I disable this option, OTA updates works, and Apple Intelligence is 100% too

Hope it helps some people.

**Comment by noobmaster69-source** [comment]
> Forgive my poor English!
> 
> I don’t know if anyone else is stuck at 99% download like me.
> 
> **The reason is I turned “Enable Internal Storage” option**
> 
> Found this problem because my OTA update is not working When I disable this option, OTA updates works, and Apple Intelligence is 100% too
> 
> Hope it helps some people.

This is interesting. The question is:

What devices are you using?
Do we only need to check 'enable internal storage' in Nugget, and then the downloading stuck issue will go away?

**Comment by chuck00000** [comment]
When you spoof your Device, of course you won't get any updates. You need to revert it, to receive updates

**Comment by hxhlb** [comment]
> > Forgive my poor English!
> > I don’t know if anyone else is stuck at 99% download like me.
> > **The reason is I turned “Enable Internal Storage” option**
> > Found this problem because my OTA update is not working When I disable this option, OTA updates works, and Apple Intelligence is 100% too
> > Hope it helps some people.
> 
> This is interesting. The question is:
> 
> What devices are you using? Do we only need to check 'enable internal storage' in Nugget, and then the downloading stuck issue will go away?

Oh, it's 16PM.
I have 13PM, but waiting for TrollStore, so it is 17.3.1
I didn't do more testing, just hope it helps some people

**Comment by DrJapan** [comment]
Not really, as your 16PM qualifies for AI natively. As your 13PM is on 17.3.1, it's not on the right OS to get AI.

**Comment by hxhlb** [comment]
> Not really, as your 16PM qualifies for AI natively. As your 13PM is on 17.3.1, it's not on the right OS to get AI.

I didn't make it clear. I am a user in China, so I need to trick the AI ​​function into turning on.

I have two devices, 13PM and 16PM
16PM After cheating, just turn off the “Enable Internal Storage” option.
13PM The system remains on 17.3.1 because I am waiting for trolls
I mean I don't have more devices to test if turning off "Enable Internal Storage" really has an effect on stuck at 99%
